pkg/collector/corechecks/snmp/snmp.go

```diff
@@ -161,30 +60,27 @@ func (c *Check) Configure(rawInstance integration.Data, rawInitConfig integratio
 		return fmt.Errorf("common configure failed: %s", err)
 	}
 
-	config, err := buildConfig(rawInstance, rawInitConfig)
+	c.config, err = checkconfig.NewCheckConfig(rawInstance, rawInitConfig)
```

Comment:
IP address should be assigned here

Reply:
I'm not sure to understand your comment. Why the IP address should be assigned here ?

Is it related to this https://github.com/DataDog/datadog-agent/blob/c28181b87b90cfd3ad8925f8c154548d5562b70d/pkg/collector/corechecks/snmp/checkconfig/config.go#L207

Reply:
This comment is related to https://github.com/DataDog/datadog-agent/pull/8971#discussion_r706064708